ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -801,197 +368,113 @@ sub update_afs_cells ( $$ ) {
 }
 
 ##########################################################################
-sub Unconfigure_Cell {
+sub Configure_Afsd_Args {
 ##########################################################################
-  unless ($NoAction) {
-   my ($self,$config)=@_;
-
-   return unless authconfig_OK($self,$authconfig);
-
-  $self->warn("Unconfigure_Cell does not work for now (authconfig broken)");
-  return;
+  my ($self,$config)=@_;
 
-   my $comm=$authconfig . " --nostart --kickstart --disableafs";
-   my $s=`$comm`;
-    if ($? >> 8) {
-      $self->error("can't run $s, no changes made");
-      return;
-    }
-    $self->OK("Unconfigured AFS client");
+  my $path = "$mypath/afsd_args";
+  if ($config->elementExists($path)) {
+      if (! $NoAction) {
+	  my $args = $config->getElement($path)->getTree();
+	  my $content;
+	  $content .= "$_:" . $args->{$_} . "\n" for sort keys %$args;
+	  my $fh = CAF::FileWriter->new("/etc/afsd.args", log => $self, backup => ".old");
+	  print $fh $content;
+	  $fh->close();
+	  $self->OK("Configured afsd.args with $content");
+      } else {
+	  $self->info("may need to update afsd.args");
+      }
   }
-  return; 
 }
 
-##########################################################################
-sub Unconfigure_firewall {
-##########################################################################
-  my ($self,$config)=@_;
+1; #required for Perl modules
 
-  if (! open(FD, "<$iptables")) {
-    $self->info("iptables will not be unconfigured: error opening file $iptables: $!");
-    return 0;
-  }
 
-  my @iptables = <FD>;
-  close(FD);
-  my @newtables;
-  my $return = 1;
-  my $found = 0;
+=head1 NAME
```

Comment:
move this to the top of the module and it's respective methods. it is much better to keep the pod close to the code